### ISSUE TITLE
Take Account for accessibilityActivationPoint

### ISSUE THREAD

**Issue by fabb** [calabash/calabash-ios#780]
`accessibilityActivationPoint` might give a different point than the middle of an element. `touch` does not take account for that and might miss the clicking area of an element, as it touches in its center.
I'm not sure whether `touch` should take account for `accessibilityActivationPoint` or not. What do you think?

**Comment by jmoody** [comment]
@fabb This is very interesting.  I am not sure what priority to give this.  Can you provide a concrete example where the activation point is different from the center?  Is the activation point defined on all UIViews?   Does it default to the center?

**Comment by fabb** [comment]
@jmoody I just learned about it too, as I tried to make our login mask usable for blind people. Our login mask is *not* table view based, but contains `UISwitch`es with Labels next to it. We have a custom view `SwitchWithLabelView` for these. For accessibility I made the whole view into an accessibility element by overriding `-isAccessibilityElement` and returning `YES`. Apart from overriding `-accessibilityLabel`, `-accessibilityValue` and `-accessibilityTraits` I also had to override `-accessibilityActivationPoint` as with VoiceOver activated, tapping on the element would not toggle the switch.
Using Calabash for automated UI tests is a bit different from having VoiceOver activated, so I'm also not sure which priority this has. As a workaround for our tests, we adapted the query to press the `descendant switch` of the given accessibility id for these `SwitchWithLabelView` elements.
